line-bot-model/src/main/java/com/linecorp/bot/model/response/NarrowcastProgressResponse.java

```diff
@@ -67,6 +69,17 @@
      */
     Long errorCode;
 
+    /**
+     * Narrowcast message request accepted time in milliseconds.
+     */
+    ZonedDateTime acceptedTime;
```

Comment:
I used ZonedDateTime here. Is there any opinion? @imasahiro

Reply:
what about using `Instant` here (for consistency)?